sdkit/models/__init__.py

```diff
@@ -1,17 +0,0 @@
-from .model_loader import (
```

Comment:
Why has this file been deleted? It's intended to create the API under `sdkit.models`.